package.json

```diff
@@ -27,6 +27,7 @@
     "history": "^4.10.1",
     "io-ts": "^2.2.10",
     "io-ts-types": "^0.5.11",
+    "jquery": "^3.5.1",
```

Comment:
I guess `bootstrap` pulls in `jquery`, even though we're only really using the CSS?  Presumably if we don't import it anywhere we're not actually paying for it in our bundle, right?

Reply:
Yeah, I think that's right. `bootstrap` has both `jquery` and `popper` (below) as peer dependencies, meaning that we're supposed to include them ourselves. However, I think that `react-bootstrap` side-steps all of the built-in JS in bootstrap (certainly I don't load it anywhere) so neither should end up in the bundle.